core/market/decentralized/src/matcher/error.rs

```diff
@@ -26,6 +26,14 @@ pub enum QueryOffersError {
     IdentityError(#[from] IdentityError),
 }
 
+#[derive(thiserror::Error, Debug)]
+pub enum QueryDemandsError {
+    #[error("Failed to get Demands. Error: {0}.")]
+    DbError(#[from] DbError),
+    #[error("Failed to list Demands based on identity. Error: {0}.")]
+    IdentityError(#[from] IdentityError),
```

Comment:
How can you get IdentityError when quering demands?

Reply:
I just copied it from offer. It's not needed - removed.